x-pack/plugins/ml/public/explorer/explorer_swimlane.js

```diff
@@ -26,8 +26,11 @@ import { mlEscape } from '../util/string_utils';
 import { mlChartTooltipService } from '../components/chart_tooltip/chart_tooltip_service';
 import { mlExplorerDashboardService } from './explorer_dashboard_service';
 import { DRAG_SELECT_ACTION } from './explorer_constants';
+import { injectI18n } from '@kbn/i18n/react';
+
+export const ExplorerSwimlane = injectI18n(class extends React.Component {
```

Comment:
use export const ExplorerSwimlane = injectI18n(class ExplorerSwimlane extends React.Component
instead of displayName